cpp/csp/cppnodes/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 add_library(baselibimpl STATIC baselibimpl.cpp)
-target_link_libraries(baselibimpl csp_core csp_engine)
+target_link_libraries(baselibimpl csp_core csp_engine m)
```

Comment:
Typo (m) ?

Reply:
`-lm` but might not be necessary anymore

Reply:
removed